front/app/components/IdeaButton/TippyContent.tsx

```diff
@@ -134,26 +89,7 @@ const TippyContent = ({ projectId, inMap, disabledReason, phase }: Props) => {
     >
       <TooltipContentIcon name="lock" ariaHidden />
       <TooltipContentText>
-        <FormattedMessage
-          {...disabledMessages[disabledReason]}
-          values={{
-            verificationLink: (
```

Comment:
Why don't we need to pass these values anymore?

Reply:
That is a very good point - totally should not have removed that

Reply:
Actually, I've just reviewed it and the reason it's removed is that in removing the `maybeNotPermitted` status which could never actually be set - there were no longer any disabled messages that had any link variables to replace